consensus/hotstuff/committees/cluster_committee.go

```diff
@@ -119,6 +120,22 @@ func (c *Cluster) Identity(blockID flow.Identifier, nodeID flow.Identifier) (*fl
 	return identity, nil
 }
 
+// IdentitiesByEpoch returns the initial cluster members for this epoch. Since clusters
+// only exist for one epoch, we don't need to check the view.
+func (c *Cluster) IdentitiesByEpoch(_ uint64, selector flow.IdentityFilter) (flow.IdentityList, error) {
```

Comment:
I think it might increase clarity if we explicitly point out that the view here is the view within the local collector cluster. Just to avoid confusion. 

same here: 
* https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/committees/cluster_committee.go#L131
* https://github.com/onflow/flow-go/blob/4de4231e43a65fe672a54aab4f84971ecac03db6/consensus/hotstuff/committees/cluster_committee.go#L139